linked_lists/web/README.md

```diff
@@ -243,12 +243,12 @@ nodeAt(index) {
   let counter = 0;
   let pointer = this.head;
 
-  while(counter < index) {
+  while(counter < index && pointer /* returns null if index > counter */) {
     pointer = pointer.next;
     counter++;
   }
 
-  return pointer;
+  return pointer.value;
```

Comment:
if pointer is `null` (or any falsy value) we will crash here. This change helps prevent a crash in the loop, but leaves the crash for out here. It could be

`return pointer && pointer.value`

but i think that's kinda ugly. Perhaps the most elegant solution is to compare the input index with `this.length` and return a nice error message? This was suggested by the students too!